src/components/AudioContext.jsx

```diff
@@ -0,0 +1,24 @@
+import { createContext, useState } from 'react';
```

Comment:
Please be consistent with whether you're using semicolons or not, and whether you're using double/single quotation marks